SampleApplications/2023/ServerlessDocumentAnalysis/infrastructure/.gitignore

```diff
@@ -0,0 +1,350 @@
+# CDK asset staging directory
```

Comment:
I think you can move the .gitignore file to the project root folder

Reply:
Good idea! Will make it happen.